governance-proposal/TSC-Charter-Draft.md

```diff
@@ -0,0 +1,103 @@
+## ***DRAFT for Public Comment***
+
+# Technical Steering Committee (TSC) Charter
+
+## Section 1. Guiding Principle.  
+
+The Node.js Foundation will operate transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to outsiders.
+
+## Section 2. Evolution of Node.js Foundation Governance.
+
+Most large, complex open source communities have both a business and a technical governance model. Node.js Foundation’s technical leadership contains both a Technical Steering Committee (“TSC”) and Maintainers for major components or subsystems. Node.js Foundation’s business leadership is instantiated in a Board of Directors (the “Board”).  
+
+This Technical Steering Committee Charter reflects a carefully constructed balanced role for the TSC and the Board in the governance of Node.js Foundation. The charter amendment process is for the TSC to propose changes using simple majority of the full TSC, the proposed changes being subject to review and approval by the Board. The Board may additionally make amendments to the TSC charter at any time, though the Board will not interfere with day-to-day discussions, votes or meetings of the TSC.
+
+## Section 3. Board’s Role in Setting Node.js Foundation’s Strategic Direction.  
+
+The Board will set the overall TSC Policy. The policy will describe the overarching scope of the Node.js Foundation initiative, Node.js Foundation’s technical vision and direction and project release expectations in the form of expected cadence and intent. The Board will use the TSC as a delegate body for governing technical implementation, individual project scope and direction while they remain within the scope and direction of the policies as described in the TSC Policy document and approved by the Board.
```

Comment:
@Fishrock123 I talked with you a bit on IRC the other day about the required order of operations. But something else I should say: we need to create structures and policies which provide the right incentives and to do that we need to understand the motivations of the people involved.

The board doesn't actually **want** to make technical decisions. They want the project to run successfully in a way that doesn't hurt their business *without* needing to involve themselves directly. Those business interests vary from company to company and that's why it's actually quite nice to have a board of them so they can figure out what the commonalities are and share any concerns they might have with the TSC (who would be foolish to ignore input from the majority of businesses supporting the platform they are building). The concerns we had that lead to creating io.js are shared by many of those companies, otherwise the foundation wouldn't be trying to reconcile the projects. Those companies are now on a board or directors which is administering this foundation and if for some reason they ever did go crazy and try to make technical decisions they'd have to convince the other members of the board on those same technical decisions. Even more importantly, they would have to do *all* of this publicly.

These "what if" concerns are the same kinds of concerns you could have about anything, even io.js. What if more than 50% of the io.js TC decided to move the project to Java? It *could* happen if they all agreed to it but we all know it's so unlikely to happen that it's not worth even considering as a possibility and the reason it's so unlikely is that there are too many incentives for all the actors involved *not* to do it.

What you don't see in these documents are the legal, marketing, and public relations efforts. That is where these companies *do* want to be more involved. And being that all of those things cost money and they are the ones putting up that money I don't see why we shouldn't let them run it. Besides, they are actually quite a bit better at doing those things than a bunch of developers are. I'm sure there will end up being collaboration with the technical side on legal matters and evangelism on the marketing because there are incentives for everyone involved to collaborate together because we all bring something important to the table, we just need a place to facilitate that collaboration and that is what the foundation is meant to be.

We aren't going to all work together because we all have guns on the table pointed at each other. We're going to work together because one set of people have peanut butter, another jelly, and another some bread and a PB&J is far more delicious than any of those things individually :)